src/app/issues/page.tsx

```diff
@@ -31,16 +31,15 @@ interface IIssueData {
 }
 
 export default function IssuesPage({}) {
-  const [issueType, setIssueType] = useState<
-    "created" | "assigned" | "mentioned"
-  >("created");
-
-  const [search, setSearch] = useState<string>(
-    `is:open is:issue author:xxxxx archived:false`
-  );
   const [issues, setIssues] = useState<IIssueData[]>(openData);
   const [issueStatus, setIssueStatus] = useState<"open" | "closed">("open");
 
+  const searchParams = useSearchParams()!;
+
+  const q = searchParams.get("q");
+
+  console.log("q ====> ", q);

```

Comment:
Remove all console.logs when finished with the branch.

Reply:
absolutely!  this is still a WIP.   This is just an attempt to think about the page structure as it pertains to search query